htdocs/install/mysql/migration/16.0.0-17.0.0.sql

```diff
@@ -149,3 +149,4 @@ UPDATE llx_c_effectif SET code='EF101-500', libelle='101 - 500' WHERE code='EF10
 
 ALTER TABLE llx_rights_def ADD COLUMN tms timestamp DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP;
 
+ALTER TABLE llx_establishment ADD COLUMN label varchar(255) NOT NULL AFTER entity;
```

Comment:
I think this was just a bad naming of column.
It may be "name".
What do you think of renaming "name" into "label" (and update code to use "label" instead) ?

Reply:
After another check no code is using the "name" column, so renaming the column is better

Reply:
After another check no code is using the "name" column, so renaming the column is better